includes/RequestWiki/SpecialRequestWiki.php

```diff
@@ -35,6 +35,7 @@ public function execute( $par ) {
 		$this->setHeaders();
 
 		$this->checkExecutePermissions( $this->getUser() );
+		$this->checkReadOnly();
```

Comment:
should be checked before permissions I think.

Reply:
Changed